src/main/java/com/ericsson/ei/subscription/InformSubscriber.java

```diff
@@ -89,13 +89,26 @@
     private JmesPathInterface jmespath;
 
     @Autowired
-    private SendHttpRequest restTemplate;
+    private HttpRequestSender restTemplate;
 
     @Autowired
     private MongoDBHandler mongoDBHandler;
 
     @Autowired
-    private SendMail sendMail;
+    private EmailSender emailSender;
+
+
+    /**
+     * This method is responsible to display the configurable application properties
+     * regarding subscription notifications.
+     */
+    @PostConstruct
+    public void init() {
+        LOGGER.debug("missedNotificationCollectionName : " + missedNotificationCollectionName);
```

Comment:
Merge these messages to one LOGGER.

Reply:
This could also be made like a collection with some pretty printing where it states that is is going to log notification settings
"--- Loggin Subscription Notification Settings ---
\nMy setting 1 ::: value ... and so on "